### DISCUSSION TITLE
GitHub actions fails because of spanish characters

### DISCUSSION THREAD

**Discussion by oliveriod** [community/community#26505] [Actions]
<p>Hi, I’m new to GitHub Actions so on my first try it just fails.</p>
<p><a href="https://github.com/oliveriod/ByblosMiPH_API/commit/e68917d9f6629b693f3b7e0098612b711e8b7d70#annotation_168268841" rel="nofollow noopener">Check failure on line 50 in ByblosMiPH_API/Startup.cs</a></p>
<p><img src="https://avatars2.githubusercontent.com/in/15368?v=4" alt="@github-actions">github-actions / build</p>
<p>ByblosMiPH_API/Startup.cs#L50</p>
<pre><code>Unexpected character '�'
</code></pre>
<p>This is line 50:</p>
<table><tbody><tr><td>
<span>var</span> <span>Par</span>á<span>metrosParaBitacora</span> <span>=</span> <span>Configuration</span>.<span>GetSection</span>(<span><span>"</span>ParámetrosParaBitácora<span>"</span></span>).<span>Get</span>&lt;<span>Par</span>á<span>metrosParaBit</span>á<span>cora</span>&gt;();</td></tr></tbody></table>
<p>The variables have accented characters.</p>
<p>The code compiles perfectly on my machine.</p>
<p>Why it fails when compiling on GitHub?</p>
<p>Is there a way to configure the use of accented characters?</p>
<p>Thanks in advance!</p>

**Comment by Yanjingzhu** [comment]
<p>Hi <a class="mention" href="/oliveriod">@oliveriod</a> , </p>
<p>Could you please share your workflow yml content here? Which type of runner do you use? Window, Ubuntu or MacOs?  Which step failed?</p>
<p>It would be better if you could share me a sample project . I could not reproduce your issue in my side.</p>
<p>Does the line 50 of Startup.cs show correctly in code page?  </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/27e168baa3a341903a7e1765cc5c512263da5509.png" data-download-href="/uploads/short-url/5GNyBnPd3aUDhhtRCZvJyNPmimd.png?dl=1" title="code.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181098082-a7178365-7c77-40b1-93af-d0e74684b453.png" alt="code.png" data-base62-sha1="5GNyBnPd3aUDhhtRCZvJyNPmimd" width="690" height="422" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/27e168baa3a341903a7e1765cc5c512263da5509_2_690x422.png, https://aws1.discourse-cdn.com/github/original/2X/2/27e168baa3a341903a7e1765cc5c512263da5509.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/27e168baa3a341903a7e1765cc5c512263da5509.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/27e168baa3a341903a7e1765cc5c512263da5509_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">code.png</span><span class="informations">1033×633 33.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by oliveriod** [comment]
<p>Hi Yanjingzhu!</p>
<p>Thanks for your reply.</p>
<p>Yes, line 50 shows up correctly.</p>
<p>I created a new very simple project that fails in the same way.</p>
<p>You can find it here:</p>
<p><a href="https://github.com/oliveriod/MiPrueba" rel="nofollow noopener">https://github.com/oliveriod/MiPrueba</a></p>
<p>Thanks again for your help.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/oliveriod">@oliveriod</a> , </p>
<p>What’s the operating system of your local work machine? Windows or Ubuntu?  </p>
<p>In your workflow , you choose ubuntu runnner to run your job. </p>
<p>I tested using windows runner, it could build the project successfully. </p>
<p>Please try  <strong>runs-on: windows-latest</strong>  . </p>
<pre><code>name: .NET Core

on:
  push:
    branches: [master]
  pull_request:
    branches: [master]

jobs:
  build:

**runs-on: windows-latest** steps:
    - uses: actions/checkout@v2
    - name: Setup .NET Core
      uses: actions/setup-dotnet@v1
      with:
        dotnet-version: 3.1.101
    - name: Install dependencies
      run: dotnet restore
    - name: Build
      run: dotnet build --configuration Release --no-restore
    - name: Test
      run: dotnet test --no-restore --verbosity normal
</code></pre>

**Comment by oliveriod** [comment]
<p>Hi Yanjingzhu!</p>
<p>You were right!</p>
<p>It works now!</p>
<p>Thank you very much!</p>
